Yoast/wordpress-seo
# Fix icon for notifications on dashboard

## Context


* Icons disappeared because of a missing class name 

## Summary


This PR can be summarized in the following changelog entry:

* Fixes the class name that adds the icon

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch
* Check if the icons are back in the notifications on the dashboard

## UI changes

## Documentation

## Quality assurance


Fixes #
